### PR TITLE
Localize country list

### PR DESCRIPTION
**Pull request by TheoChevalier**
@ScottDowne It’s throwing this https://github.com/marcbachmann/localized-countries/blob/master/browser.js#L6-L7 but I don’t understand where I’m not passing what it needs :/

### CODE REVIEW

**Comment by TheoChevalier** [comment]
Ok, found a workaround, @ScottDowne does that look good enough to you?

File: src/components/signup-form.js [hunk 1]
```diff
@@ -78,6 +78,19 @@ var Signup = React.createClass({
...
   },
   render: function() {
+    var siteCountries = ['AT', 'BE', 'BG', 'HR', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'DE', 'GR', 'HU', 'IE', 'IT', 'LV', 'LT', 'LU', 'MT', 'NL', 'PL', 'PT', 'RO', 'SK', 'SI', 'ES', 'SE', 'GB'];
+    var supportedLocales = require('../../node_modules/localized-countries/languages.json');
+    var selectedLocale = 'en';
+    var targetLocale = this.context.intl.locale.replace(/-/g , "_");
+    if (supportedLocales.indexOf(targetLocale) >= 0) {
+      selectedLocale = targetLocale;
+    }
+    var localizedCountriesAll = require('../../node_modules/localized-countries/data/' + selectedLocale + '.json');
```
Comment (ScottDowne) [inline src/components/signup-form.js]: How is this working? I thought we couldn't do require calls with dynamic values?

Comment (ScottDowne) [inline src/components/signup-form.js]: I can confirm though, it IS working.

Comment (TheoChevalier) [inline src/components/signup-form.js]: Hm, didn’t know we couldn’t (but there’s a lot of things I don’t know)

Comment (ScottDowne) [inline src/components/signup-form.js]: I think it works because of https://webpack.github.io/docs/context.html

**Pull request closed without merging**